fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeChecker.java

```diff
@@ -1013,14 +1014,22 @@ public Void visitShowRoutineLoadStatement(ShowRoutineLoadStmt statement, Connect
             return null;
         }
 
+        @Override
+        public Void visitShowRoutineLoadTaskStatement(ShowRoutineLoadTaskStmt statement, ConnectContext session) {
+            String db = statement.getDbFullName();
+            if (!GlobalStateMgr.getCurrentState().getAuth().checkDbPriv(session, db, PrivPredicate.SHOW)) {
```

Comment:
PrivPredicate.LOAD on table is required in original logic, better to keep the same with it.
you can refer to ShowExecutor.handleShowRoutineLoadTask.